plenum/test/replica/test_update_watermarks_api.py

```diff
@@ -48,6 +50,7 @@ def test_propagate_primary_non_Master_watermarks_not_maxsize_if_is_primary(repli
     assert replica.H == 100 + tconf.LOG_SIZE
 
 
+@pytest.mark.skip(reason="For now, we don't drop the ppSeqNo and watermarks")
```

Comment:
It makes sense to edit the test instead of skipping it:
- it looks like we still change watermakrs
- it may make sense to check that we DON'T drop ppSeqNo (and hence `last_ordered_3pc `)
- it may make sense to add other checks (what is expected in `_catchup_clear_for_backup` call)

Reply:
ok